code-challenges.test.js

```diff
@@ -14,17 +16,36 @@
 // Reminder: The test will call your function
 
 // --------------------1) Create a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
-
-// a) Create a test with an expect statement using the variables provided.
-
 var secretCodeWord1 = "Lackadaisical"
 // Expected output: "L4ck4d41s1c4l"
 var secretCodeWord2 = "Gobbledygook"
 // Expected output: "G0bbl3dyg00k"
 var secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
-
+// a) Create a test with an expect statement using the variables provided.
+Describe("codedMessage",() =>{
+    test("takes in a string and returns a coded message",() =>{
+        let secretCodeWord1 = "Lackadaisical"
+        let secretCodeWord2 = "Gobbledygook"
+        let secretCodeWord3 = "Eccentric"
+        expect(codedMessage(secretCodeWord1).toEqual(["L4ck4d41s1c4l"]))
+        expect(codedMessage(secretCodeWord2).toEqual(["G0bbl3dyg00k"]))
+        expect(codesMessage(secretCodeWord3).toEqual(["3cc3ntr1c"]))
+        // When i ran my test it gave me a ReferenceError. Describe is not defined. 
+    })
+})
+// create a function called codedMessage 
+// Paramater - string
+// I want to create a new string and replace the letters with numbers using the .replace method. 
+// return newStr
+
+const codedMessage =(string)=> {
+   let newStr = string.replace('a',4, 'e',3, 'i', 1, 'o', 0)
```

Comment:
.replace can only take two arguments-- Great logic you'd just have to do the same thing for each vowel you want manipulated.